src/components/navigation/Tabs/Tabs.tsx

```diff
@@ -4,22 +4,16 @@ import { Box } from '../../structure'
 import { useStyles } from '../../../theme'
 import { Tab } from './Tab/Tab'
 
-export interface TabDescriptor {
-  /**
-   * ID of the tab.
-   */
-  id: string
-  /**
-   * Text of the tab.
-   */
-  label: string
-}
+/**
+ * Text of the tab.
+ */
+export type TabDescriptor = string
 
 export interface TabsProps {
   /**
    * List of tabs.
    */
-  tabs: [TabDescriptor, ...TabDescriptor[]]
+  tabs: TabDescriptor[]
```

Comment:
Since this ended up being a string, should we just opt for `string[]` (to make the documentation simpler)?

Reply:
Changed it for `string[]` and removed `TabDescriptor` :slightly_smiling_face: